### DISCUSSION TITLE
how to understand this code sequence

### DISCUSSION THREAD

**Discussion by ItsPiyusha** [TheAlgorithms/Python#8041] [General]
What file is to be read first?

**Comment by CaedenPH** [comment]
> What file is to be read first?

These files are not connected with each other; Each file is standalone. If you want to see how an algorithm is implemented, you could find this file through the DIRECTORY.md, located in the root directory.

**Reply by Nomine-Nihil** [reply]
You sure? Maybe I'm wrong but I'm not seeing how they're standalone yet I can't run it either to read..lol halp

**Reply by CaedenPH** [reply]
> You sure? Maybe I'm wrong but I'm not seeing how they're standalone yet I can't run it either to read..lol halp
> 

What exactly do you mean? Each file is independent of each other and if you run it, sometimes there will be an output with some test data, but not always

**Comment by Vipin654321** [comment]
Just try to understand the flow
